posts/call-for-testing-trusty-snaps.md

```diff
@@ -0,0 +1,63 @@
+There is a huge announcement coming: snaps now run in Ubuntu 14.04 Trusty Tahr.
```

Comment:
You should unify the use of Trusty vs trusty. I would rather go for Trusty, but I'm not a native speaker myself.